crates/yttrium/src/chain_abstraction/api/mod.rs

```diff
@@ -10,18 +10,39 @@ uniffi::custom_type!(Address, String, {
     lower: |obj| obj.to_string(),
 });
 
+#[cfg(feature = "uniffi")]
+uniffi::custom_type!(U256, String, {
+    try_lift: |val| Ok(val.parse()?),
+    lower: |obj| obj.to_string(),
+});
+
+#[cfg(feature = "uniffi")]
+uniffi::custom_type!(U64, String, {
+    try_lift: |val| Ok(val.parse()?),
+    lower: |obj| obj.to_string(),
+});
+
+#[cfg(feature = "uniffi")]
+uniffi::custom_type!(Bytes, String, {
+    try_lift: |val| Ok(val.parse()?),
+    lower: |obj| obj.to_string(),
+});
+
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
 pub struct Transaction {
     pub from: Address,
     pub to: Address,
-    pub value: String,
-    pub gas: String,
-    pub gas_price: String,
```

Comment:
this change affects swift bindings, so please notify me of changes like this too

Reply:
I think this should fail CI then?

Reply:
agree but unfortunately i don't have tests for types decoding and not gonna add them as we are going to change ffi lib

Reply:
I added some `git diff`s at the end of the CI, hopefully this will catch this